### PR TITLE
Reformating the recurring event page

### PR DESCRIPTION
**Pull request by vazquez**
Redesigning the page as seen in https://github.com/mozilla/mozmaker/issues/48

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -18,13 +18,17 @@
...
+        <a>firefox-orange</a>
+        <a>market-orange</a>
+        <a>flame-yellow</a>
+        <a>gecko-green</a>
+        <a>summit-teal</a>
+        <a>mobile-blue<a>
+        <a>developer-blue</a>
+        <a>nightly-blue</a>
+        <a>aurora-purple</a>
+        <a>bikeshed-magenta</a>
```
Comment (gvn) [inline src/index.html:31]: Nice. :laughing:

**Comment by gvn** [comment]
Something weird is going on for me with the layout:

screen shot 2016-03-16 at 2 38 57 pm

The Etherpad embed is wonky...

**Comment by gvn** [comment]
Ricardo: Fixed!

---

Something is funky with your editor because you've got mixed soft and hard tabs going on. Can you fix the HTML so that it's only 2 space indentation?

screen shot 2016-03-16 at 2 41 12 pm

Sorry, this is hard to see, but if you turn on invisibles in your editor it should help you visualize it. You should just see dots for the indentation...

File: src/templates/curriculum-workshop/style.scss [hunk 2]
```diff
@@ -1,5 +1,84 @@
...
+    border-right-width: 0;
+    border-bottom-width: 3px;
+    border-left-width: 0;
+
+    border-style: solid;
+
+    border-top-color: rgba(0, 0, 0, 0);
+    border-right-color: rgba(0, 0, 0, 0);
+    border-bottom-color: rgba(0, 0, 0, 0);
+    border-left-color: rgba(0, 0, 0, 0);
```
Comment (gvn) [inline src/templates/curriculum-workshop/style.scss]: Could you do the previous 4 with just `border-color: $black;` ?

File: src/templates/curriculum-workshop/style.scss [hunk 3]
```diff
@@ -1,5 +1,84 @@
...
+.tab-container {
+
+  input {
+    display: none;
+    &:checked + label {
+      border-top-width: 3px;
+      border-right-width: 3px;
+      border-bottom-width: 0;
+      border-left-width: 3px;
+      border-style: solid;
```
Comment (gvn) [inline src/templates/curriculum-workshop/style.scss]: Isn't solid the default?

**Comment by gvn** [comment]
Tabs also don't seem to be working:

screen shot 2016-03-16 at 2 50 23 pm

**Comment by gvn** [comment]
I'm seeing these display issues in Chrome and FF.

File: src/templates/main.scss [hunk 4]
```diff
@@ -56,6 +56,31 @@
...
 
+  .tab-container {
+
+    input {
+      &:checked + label {
+        color: $primary-dark;
+        border-top-color: $primary-light;
+        border-right-color: $primary-light;
+        border-bottom-color: $primary-light;
+        border-left-color: $primary-light;
```
Comment (gvn) [inline src/templates/main.scss]: Could do just `border-color: $primary-light;`

**Comment by gvn** [comment]
Sorry, gonna need some fixups before we can land. :/

computar

File: src/templates/curriculum-workshop/style.scss (hunk 2 above)
Comment (vazquez) [inline src/templates/curriculum-workshop/style.scss]: I need the transparency. But `border-color: rgba(0,0,0,0);` will work.

File: src/templates/curriculum-workshop/style.scss (hunk 3 above)
Comment (vazquez) [inline src/templates/curriculum-workshop/style.scss]: Hahaha, you're right!

File: src/templates/main.scss (hunk 4 above)
Comment (vazquez) [inline src/templates/main.scss]: Done :+1:

**Comment by vazquez** [comment]
I'm not seeing any display issues in Chrome, FF and FF DevEdition :confused: 

Maybe, just maybe, could it be because of the PR I just put up on mozmaker repo? 
https://github.com/mozilla/mozmaker/pull/56

**Comment by gvn** [comment]
Ah, ok. I didn't know you had `mozmaker` npm linked so that is why it was broken for me. :open_mouth: 

Let's get that PR landed first, and then I think this will be safe to land.

**Comment by vazquez** [comment]
Perfect :+1:

**Comment by gvn** [comment]
I think we should do the tabs with flexbox to avoid this happening when lines break:

screen shot 2016-03-17 at 9 50 03 am

and:

screen shot 2016-03-17 at 9 51 46 am

**Comment by vazquez** [comment]
Okay I'll try it out. Make sure that you've pulled the most recent edits, I have made visual changes to those tabs.

**Comment by gvn** [comment]
Ah, k. Still seeing it, but less severe:

screen shot 2016-03-17 at 9 55 24 am

**Comment by gvn** [comment]
R+!

dembrows

**Comment by vazquez** [comment]
Hahahaha, that gif made it all worthwhile.

**Pull request merged**